safe_transaction_service/history/services/transaction_service.py

```diff
@@ -227,12 +227,11 @@ def get_all_tx_hashes(
             ERC721Transfer.objects.to_or_from(safe_address)
             .exclude(ethereum_tx__in=multisig_and_module_hashes)
             .annotate(
-                execution_date=F("ethereum_tx__block__timestamp"),
-                created=F("ethereum_tx__block__timestamp"),
-                block=F("ethereum_tx__block_id"),
```

Comment:
just to fully understand the old code: would `ethereum_tx__block_id` be the same as `ethereum_tx__block__number`, just with an additional internal db lookup?

Reply:
Yes. Well, Django is clever enough to replace `ethereum_tx__block__number` to `ethereum_tx__block_id`